### PR TITLE
Update linting

### PR DESCRIPTION
**Pull request by mabhub**

### CODE REVIEW

File: src/modules/Map/InteractiveMap/InteractiveMap.test.js [hunk 1]
```diff
@@ -1,3 +1,5 @@
+/* eslint-disable max-classes-per-file */
```
Comment (dtrucs) [inline src/modules/Map/InteractiveMap/InteractiveMap.test.js:1]: oh god, lazy boys (including me) would not like this rule :D

File: src/modules/Forms/Controls/Select/Select.js [hunk 2]
```diff
@@ -125,8 +125,7 @@ export class Select extends React.Component {
             onQueryChange={this.handleQueryChange}
             initialContent={(values && values.length > 250)
               ? <MenuItem disabled text={translate('terralego.forms.controls.select.select_placeholder')} />
-              : undefined
-            }
+              : undefined}
```
Comment (dtrucs) [inline src/modules/Forms/Controls/Select/Select.js:128]: Not the subject of the PR, but  `&&` seems more appropriate than a conditional ternary

File: src/modules/Map/InteractiveMap/InteractiveMap.test.js (hunk 1 above)
Comment (mabhub) [reply src/modules/Map/InteractiveMap/InteractiveMap.test.js:1]: This is a test file.
If you want to group tests by concern, you have to allow exceptions to this rule. (Tests requires creating temporary classes)

File: src/modules/Forms/Controls/Select/Select.js (hunk 2 above)
Comment (mabhub) [reply src/modules/Forms/Controls/Select/Select.js:128]: Right, but as you said, I try to not embed in this PR anything else than linting ruleset upgrade.

**Pull request merged**